v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint needs to be made:
+
+```typescript
+POST https://security-us-east-1.aws.supertokens.io/v1/security
+```
+
+You can view the HTTP API reference for this endpoint [here](#somelinkhere).
+
+### Retrieving the Request ID
+
+The reuqest ID that has been passed from the frontend is available in the `requestId` property of the input body from each API override.
+
+Below is an example of how to retrieve the request ID from the input body:
+
+```typescript
+const generatePasswordResetTokenPOST = async function (input) {
+    // ... your code here ...
+    const requestId = (await input.options.req.getJSONBody()).requestId;
+    // ... your code here ...
+}
+```
```

Comment:
Need to mention that for emailpassword sign up, sign in and password reset, we need to enforce that the request id is present, and if not, send a 400 error (you can show the code snippet for this as its done in emialpassword.init on the backend).

The reason we throw a 400 error is to prevent a case where one can modify the frontend request to not send a requestId, and in that case these APIs will skip doing bot detection, and we dont want to skip it

Reply:
Done. Will add the snippet below